.Lib9c.Tests/Action/ClaimItemsTest.cs

```diff
@@ -0,0 +1,222 @@
+    using Serilog;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class ClaimItemsTest
+    {
+        private readonly IAccount _initialState;
+        private readonly Address _signerAddress;
+
+        private readonly TableSheets _tableSheets;
+        private readonly List<Currency> _currencies;
+        private readonly List<int> _itemIds;
+
+        public ClaimItemsTest(ITestOutputHelper outputHelper)
+        {
+            Log.Logger = new LoggerConfiguration()
+                .MinimumLevel.Verbose()
+                .WriteTo.TestOutput(outputHelper)
+                .CreateLogger();
+
+            _initialState = new MockStateDelta();
+
+            var sheets = TableSheetsImporter.ImportSheets();
+            foreach (var (key, value) in sheets)
+            {
+                _initialState = _initialState
+                    .SetState(Addresses.TableSheet.Derive(key), value.Serialize());
+            }
+
+            _tableSheets = new TableSheets(sheets);
+            _itemIds = _tableSheets.CostumeItemSheet.Values.Take(3).Select(x => x.Id).ToList();
+            _currencies = _itemIds.Select(id => Currency.Legacy($"Item_T_{id}", 0, minters: null)).ToList();
+
+            _signerAddress = new PrivateKey().ToAddress();
+
+            var context = new ActionContext();
+            _initialState = _initialState
+                .MintAsset(context, _signerAddress, _currencies[0] * 5)
+                .MintAsset(context, _signerAddress, _currencies[1] * 5)
+                .MintAsset(context, _signerAddress, _currencies[2] * 5);
+        }
+
+        [Fact]
+        public void Serialize()
+        {
+            var states = GenerateAvatar(_initialState, out var avatarAddress1);
+            GenerateAvatar(states, out var avatarAddress2);
+
+            var action = new ClaimItems(new List<(Address, List<FungibleAssetValue>)>
+                {
+                    (avatarAddress1, new List<FungibleAssetValue> { _currencies[0] * 1, _currencies[1] * 1 }),
+                    (avatarAddress2, new List<FungibleAssetValue> { _currencies[0] * 1 }),
+                });
+            var deserialized = new ClaimItems();
+            deserialized.LoadPlainValue(action.PlainValue);
+
+            Assert.Equal(action.ClaimData[0].address, deserialized.ClaimData[0].address);
+            Assert.Equal(
+                action.ClaimData[0].fungibleAssetValues[0].Currency.Ticker,
+                deserialized.ClaimData[0].fungibleAssetValues[0].Currency.Ticker);
+            Assert.Equal(
+                action.ClaimData[0].fungibleAssetValues[1].Currency.Ticker,
+                deserialized.ClaimData[0].fungibleAssetValues[1].Currency.Ticker);
+
+            Assert.Equal(action.ClaimData[1].address, deserialized.ClaimData[1].address);
+            Assert.Equal(
+                action.ClaimData[1].fungibleAssetValues[0].Currency.Ticker,
+                deserialized.ClaimData[1].fungibleAssetValues[0].Currency.Ticker);
```

Comment:
I guess it can directly check `FAV`s equality because FAV implement `IEquatable<T>` interface.

https://github.com/planetarium/libplanet/blob/7235c0f35fd8e586a91fbad87d43bc881610fb58/Libplanet.Types/Assets/FungibleAssetValue.cs#L25-L28